src/main/java/com/github/dikhan/pagerduty/client/events/PagerDutyEventsClient.java

```diff
@@ -94,6 +96,12 @@ public EventResult resolve(ResolveIncident resolve) throws NotifyEventException
         return eventResult;
     }
 
+    public EventResult sendEvent(Incident incident) throws NotifyEventException {
```

Comment:
I was thinking of removing trigger, acknowledge, and resolve method altogether and replace it with sendEvent.  Since the enforcement of the JSON structure is already checked by user constructing TriggerIncident, AcknowledgeIncident, and ResolveIncident, the client should simply be the facilitator of sending these incidents.